sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -19,12 +20,14 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public class TextAnalyticsClient
```

Comment:
Currently, this class is a pure pass-through; it has responsibility for creating the service client that corresponds to the requested version, and nothing else.   This was chosen primarily because it was the most expedient path forward.   In the future, we could consider pulling some common functionality here rather than delegating to the service clients and duplicating code.  For example:

- Parameter validation
- Doing the transforms/packaging that used the private helpers previously in the `Common` section

The trade-off for centralizing that logic would be that we lose the freedom to have logic specific to the service versions - which may or may not be desirable.   In the longer term, since the Legacy API is frozen, it may be a better pattern to localize logic being changed to the Language Service client.   Something to consider.

Reply:
I have to say, I love how this class looks :)